Rohit/PostgresAssignment/controller/class.ts

```diff
@@ -0,0 +1,53 @@
+import { Context } from "vm";
+
+const cls = require('../sql/class');
+
+interface classesinfo {
+    cls_id: number;
+    st_id: number;
```

Comment:
studentId, keep expandend names. Please try to do this changes for all variable, interface and function